tests/test_inference_client.py

```diff
@@ -136,7 +136,7 @@ def test_document_question_answering(self) -> None:
     def test_feature_extraction(self) -> None:
         embedding = self.client.feature_extraction("Hi, who are you?")
         self.assertIsInstance(embedding, np.ndarray)
-        self.assertEqual(embedding.shape, (8, 768))
+        self.assertEqual(embedding.shape, (1, 8, 768))
```

Comment:
Would it be possible to have a second test with a sentence transformers model?

Reply:
Good idea, I just added a small test case in https://github.com/huggingface/huggingface_hub/pull/1648/commits/d9c545d5661abe8aae531d6c06b4d700a5a80830. Should be enough.